addons/mail/static/src/components/dialog/dialog.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="mail.Dialog" owl="1">
         <t t-if="dialog">
-            <div class="o_Dialog" t-attf-class="{{ className }}" t-att-style="dialog.style" t-ref="root">
+            <div class="o_Dialog fixed-top bottom-0 d-flex" t-attf-class="{{ className }}" t-att-style="dialog.style" t-ref="root">
```

Comment:
It's a bit confusing to make use of `fixed-top` and then add `bottom-0`.

Looks like `fixed-top` is intended for a sort of sticky navbar, but it's hacked for showing a modal/dialog by enriching it with `bottom-0`.

Reply:
Indeed. I used `bottom-0` to include the syntax of `bottom: 0` from the SCSS, since `fixed-top` does not mention any position in the bottom. However, I think it is more logical to use in this case `h-100` with `fixed-top` and `d-flex`.

What do you think?

Reply:
It's a small detail regardless. `bottom-0` lets you think it terms of bottom borders snapped to viewport bottom, whereas `h-100` let you think of it using whole height of viewport (and consequently of top border being "snapped" top top of viewport, it's equivalent to bottom border being "snapped" to bottom of viewport).

I have a slight preference to `bottom-0` just so the whole reasoning is consistent and it's just about "snapped" borders.

Anyway it's a small details, and both are logically equivalent.